Public/Src/Cache/ContentStore/Vfs/Provider/ActiveEnumeration.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace BuildXL.Cache.ContentStore.Vfs.Provider
+{
+    using Utils = Microsoft.Windows.ProjFS.Utils;
+
+    internal class ActiveEnumeration
+    {
+        private readonly IEnumerable<VfsNode> fileInfos;

```

Comment:
`_fileInfos` ?

Reply:
I'm leaving the styling mostly unchanged to ensure better comparison with original sample where this code came from.

---
In reply to: [298336392](https://github.com/microsoft/BuildXL/pull/519#discussion_r298336392) [](ancestors = 298336392)